sources/describe.py

```diff
@@ -0,0 +1,138 @@
+import logging
+
+from Parser.ArgParser import ArgParser
+from Parser.CSVParser import CSVParser
+from math import sqrt, floor, ceil
+
+
+logging.getLogger().setLevel(logging.INFO)
+
+
+class Describe(CSVParser):
```

Comment:
CSVParser have to inherite from Describe, we could use describe in CSVParser when we want to show data while parsing file or visualise some

Reply:
I've tried, i let you check the last push.